DEMO.md

```diff
@@ -15,6 +15,7 @@
 2. Verify robot controller
   Run the robot execution node and verify that the robot arm and rail joint are moved to the home position
   ```
+  pip install kitchen
```

Comment:
I believe that I only got the error about missing the kitchen package when I tried to run the test_robot_execution.py script, that's why I put the installation of it prior. I think this would probably be more appropriate in the installation section of the main README if its required to run the base application (with or without the CNN)